barretenberg/cpp/src/barretenberg/vm2/common/standard_affine_point.hpp

```diff
@@ -0,0 +1,63 @@
+#pragma once
+
+#include <cstdint>
+
+namespace bb::avm2 {
+
+template <typename AffinePoint> class StandardAffinePoint {
+  public:
+    using CoordinateField = AffinePoint::Fq;
```

Comment:
Can we rename this to BaseField to keep in line with the naming used in the curves description